man/kpatch.1

```diff
@@ -35,6 +35,9 @@ info <module>
 list
        list installed patch modules
 
+signal
+	signal/poke any process stalling the current patch transition
```

Comment:
nitpick ... `% man -l man/kpatch.1` looks like this:
```
       list
              list installed patch modules

       signal      signal/poke any process stalling the current patch transition

       version
              display the kpatch version
```
My guess is that is man wants spaces and not tabs here.